src/kvstore/Part.cpp

```diff
@@ -70,6 +70,13 @@ void Part::asyncPut(folly::StringPiece key, folly::StringPiece value, KVCallback
         });
 }
 
+void Part::asyncAppendMultiValue(std::string&& multiValues, KVCallback cb) {
```

Comment:
maybe it's not the multi value is raw value ..

Reply:
will rename it to 'batch'

Reply:
Seems Good Name